src/ripple/app/main/Application.cpp

```diff
@@ -1254,6 +1255,12 @@ class ApplicationImp : public Application, public BasicApp
         return maxDisallowedLedger_;
     }
 
+    virtual std::optional<uint256>
```

Comment:
I would return a `const std::optional<uint256>&` instead of a copy

Reply:
Good idea.